admin/test/indexes/TagPagesTest.scala

```diff
@@ -7,12 +7,22 @@ import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.DoNotDiscover
 import test.WithTestExecutionContext
+import TagPages._
 
-@DoNotDiscover class TagPagesTest extends AnyFlatSpec with Matchers with WithTestExecutionContext with ScalaFutures {
```

Comment:
The [`@DoNotDiscover`](https://www.scalatest.org/scaladoc/3.0.5/org/scalatest/DoNotDiscover.html) annotation (added with https://github.com/guardian/frontend/pull/6028 in September 2014) meant that `TagPagesTest` was actually not getting run in CI- it was never getting run unless it was specifically requested. It looks like this was [probably unintentional](https://github.com/guardian/frontend/pull/6028#issuecomment-56163278) - I'm reinstating these tests!

The `WithTestExecutionContext` & `ScalaFutures` traits have been superfluous since https://github.com/guardian/frontend/pull/25176 removed the need for the `TagPages` class to use `Future`s.